### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by pgrimaud**
Hello,

This PR will fix : 

- 2 typos in `README.md`
- 1 typo in `docs/release.md`
- 1 typo in `make/lint.mk`
- 1 typo in `pkg/controller/config/config_controller.go`
- 2 typos in `pkg/utils/transform/transform.go`

Cheers! :robot:

### CODE REVIEW

**Comment by nikhil-thomas** [comment]
@pgrimaud Thank you for this patch.
At present, the ci config is a bit flaky. We are working on a fix.

**Comment by sthaha** [comment]
@pgrimaud thank you for the patch. Could you please try rebasing?

File: pkg/utils/transform/transform.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func InjectLabel(key, value string, overwritePolicy OverwritePolicy, kinds ...st
 		labels[key] = value
 		err = unstructured.SetNestedStringMap(u.Object, labels, "metadata", "labels")
 		if err != nil {
-			return fmt.Errorf("error updateing labes for %s:%s, %s", kind, u.GetName(), err)
+			return fmt.Errorf("error updating labes for %s:%s, %s", kind, u.GetName(), err)
```
Comment (sthaha) [inline pkg/utils/transform/transform.go:119]: also `labels` :D we were on a typo spree here

Comment (pgrimaud) [reply pkg/utils/transform/transform.go]: Nice catch, I fixed it too

**Comment by pgrimaud** [comment]
@sthaha I rebase my branch from upstream 👍

**Pull request merged**